erpnext/stock/doctype/stock_ledger_entry/stock_ledger_entry.py

```diff
@@ -38,6 +38,11 @@ def on_submit(self):
 		self.check_stock_frozen_date()
 		self.actual_amt_check()
 
+		if self.batch_no:
+			batch = frappe.get_doc("Batch", self.batch_no)
+			batch.calculate_batch_qty()
```

Comment:
this should be only upto the date of the SLE

Reply:
batch qty on the dashboard should not be date dependent right? Not sure what you mean here